marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/matcher-impl/matcher-impl.xqy

```diff
@@ -303,7 +303,7 @@ declare function match-impl:find-document-matches-by-options(
           $queries
         return
           if (fn:count($queries) gt 1) then
-            cts:and-query($queries)
+            cts:or-query($queries)
```

Comment:
@ryanjdew I don't know enough to know the impact here. Thoughts?

I see that the code prior to the latest change was this:

```
return
        if (fn:count($queries) gt 1) then
          helper-impl:group-queries-by-scope($queries, cts:and-query#1)
        else
          $queries
```

And is now this:

```
return
        let $queries := if (fn:count($queries) gt 1) then
          helper-impl:group-queries-by-scope($queries, cts:and-query#1)
        else
          $queries
        return
          if (fn:count($queries) gt 1) then
            cts:and-query($queries)
          else
            $queries
```

So this looks like a new line of code. I just don't know whether "or" or "and" is the right choice here.